juju_spell/utils.py

```diff
@@ -15,9 +15,17 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 """Utilities for JujuSpell."""
+import logging
 import secrets
 from collections import defaultdict
-from typing import Dict, Iterable, List
+from pathlib import Path
+from typing import Any, Dict, Iterable, List
+
+import yaml
+
+from juju_spell.exceptions import JujuSpellError
+
+logger = logging.getLogger()
```

Comment:
It's better to use new logger, which will inherit from root rather than using root logger.
```suggestion
logger = logging.getLogger(__name__)
```